Upgraded packages to dotnetcore 3.0

aflegel opened the pull request:
modified the tests to reference list tuples instead of aliased
Modified the query for searches to run faster.

alexanderboutinflegelphl committed:
Upgraded packages to dotnetcore 3.0
modified the tests to reference list tuples instead of aliased
Modified the query for searches to run faster.

alexanderboutinflegelphl committed:
Improved the Roll controller accessors
Improved the Roll controller tests to catch more false positives

ChrisDusyk commented on Visualizer/Startup.cs:
```diff
@@ -21,7 +22,7 @@ public Startup(IConfiguration configuration)
 		{
 			services.AddCors();
-			services.AddMvc();
+			services.AddMvc();//.AddNewtonsoftJson();
```
Clean up comments?

ChrisDusyk commented on Visualizer/Startup.cs:
```diff
@@ -54,7 +55,11 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
-			app.UseMvc(routes => { routes.MapRoute("default", "{controller=Home}/{action=Index}/{id?}"); });
+			app.UseRouting();
+
+			app.UseEndpoints(endpoints => {
```
That's neat!

ChrisDusyk reviewed (approved):
I like the refactor for the pool accessors. Looks like a pretty painless upgrade.

alexanderboutinflegelphl committed:
Split the Database Fixture from the tests
Sleaned up the Startup
Migrated the OutcomeAnalysis class into the static pool seed
stopped exposing unnecessary seed methods
renamed seed functions for clarity

alexanderboutinflegelphl committed:
Test Common readability improvements

aflegel committed:
Package updates, loggin changes

alexanderboutinflegelphl committed:
Added a health check endpoint
converted controllers to api controllers
Simplified the roll result view model

alexanderboutinflegelphl committed:
fixed an oopsie
